docs/source/configuration.rst

```diff
@@ -244,6 +246,22 @@ Options reference
      - Time in seconds to wait between two consecutive RPC requests for new events.
        The value applies only to chain NAME, taking precedence over the global poll period.
 
+   * - ::
+
+        min-source-balance = ETH
+
+     - Minimum ETH balance on source chain to fill requests on target chain..
+       The value applies to all chains that don't have the chain-specific min-source-balance defined.
+       Default: ``0.1``.
+
+   * - ::
+
+        [chains.NAME]
+        poll-period = ETH
+
+     - Minimum ETH balance on source chain to fill requests on target chain.
```

Comment:
Nitpick: I would make this slightly clearer, e.g.
```
     - Minimum ETH balance on chain ``NAME`` to fill requests originating from it.
```

Reply:
done